pkg/sql/set_cluster_setting.go

```diff
@@ -617,6 +617,28 @@ func setVersionSetting(
 			return err
 		}
 		return db.Txn(ctx, func(ctx context.Context, txn isql.Txn) error {
+			// Run the version bump inside the upgrade as high priority, since
+			// lease manager ends up reading the version row (with high priority)
+			// inside the settings table when refreshing leases. On larger clusters
+			// or with a large number of descriptors it's possible for normal
+			// transactions to be starved  (#95227).
```

Comment:
Could we add some reference numbers to "large" here? Doesn't have to be exact, just within the ballpark as large is quite ambiguous.

The wording here feels a bit difficult to digest, in particular, "normal transactions" made me think about non-system transactions but I think the intended meaning is non-high priority transactions.

What would you think about (With appropriate line wrapping and appropriate reference numbers):
```suggestion
			// On clusters with a large number of descriptors (> 10k) or nodes (>1k), non-priority transactions reading the version row can be staved. This is due to the lease manager reading the version row at high priority (#95227).
			// Run the version bump inside the upgrade as a high priority txn to avoid being bullied out by the lease manager.
```